lib/experimental/webfonts.php

```diff
@@ -61,8 +61,6 @@ function wp_webfonts() {
  * @param array $webfonts Webfonts to be registered.
  *                        This contains an array of webfonts to be registered.
  *                        Each webfont is an array.
- *                        See {@see WP_Webfonts_Registry::register()} for a list of
- *                        supported arguments for each webfont.
  */
 function wp_register_webfonts( array $webfonts = array() ) {
```

Comment:
The array of webfonts is required, not optional. Remove the default value.
```suggestion
function wp_register_webfonts( array $webfonts ) {
```

Reply:
Resolved in https://github.com/WordPress/gutenberg/pull/39559/commits/27e7da71a053c7e21c91d83935fbb1fd460f376b.